tests/models/test_boundary_condition.py

```diff
@@ -113,57 +113,97 @@ def test_boundary_condition_mixin(t_end: int):
 """Here follows mixins related to testing of Robin limit cases, and eventually the test itself. """
 
 
-class BCValues:
-    """Set boundary values for momentum balance and mass and energy balance."""
+class BCValuesLeftSide:
+    """Boundary values for primary variables on west and east boundary.
+
+    Used for:
+    * Momentum balance
+    * Mass and energy balance.
+
+    """
+
+    def not_dir_inds(self, g):
+        """Indices for the non-Dirichlet boundaries for test.
+
+        The Robin limit case test tests setups with Dirichlet on west/east boundaries,
+        and Robin approximating Dirichlet on the north/south boundaries. This method
```

Comment:
Wouldn't it be right to say that we use Robin approximating either Dirichlet or Neumann?

Reply:
And do you think it would be slightly easier to understand this method if we rename it to rob_inds?

Reply:
I renamed to rob_inds, as I agree that can make things  more understandable. Might still be a bit "weird" that rob_inds is used in the Dirichlet reference class, but I think the upsides of changing the name are big enough to accept that slight "weirdness".